### PR TITLE
Updated version numbers for names.co.uk & Register365

### PR DESCRIPTION
**Pull request by alexmace**
Based on platform updates applied today.

### CODE REVIEW

File: _data/shared_hosts.yml [hunk 1]
```diff
@@ -156,10 +156,10 @@
...
 - name: "names.co.uk"
   url: "https://www.names.co.uk/web-hosting/"
-  php54: 35
-  php55: 19
-  php56: 3
-  default: 5.5.19
+  php54: 36
+  php55: 30
+  php56: 4
+  default: 5.5.20
```
Comment (GrahamCampbell) [inline _data/shared_hosts.yml]: ?

Comment (GrahamCampbell) [inline _data/shared_hosts.yml]: really?

File: _data/shared_hosts.yml [hunk 2]
```diff
@@ -194,10 +194,10 @@
...
 - name: "Register365"
   url: "https://www.register365.com/web-hosting/"
-  php54: 35
-  php55: 19
-  php56: 3
-  default: 5.5.19
+  php54: 36
+  php55: 30
+  php56: 4
+  default: 5.5.20
```
Comment (GrahamCampbell) [inline _data/shared_hosts.yml]: really?

File: _data/shared_hosts.yml (hunk 1 above)
Comment (philsturgeon) [inline _data/shared_hosts.yml]: Any reason for your doubts?

Comment (GrahamCampbell) [inline _data/shared_hosts.yml]: why would they have a default exactly 10 patch versions behind their other version? I thought 5.5.19 might have been intentional.

Comment (GrahamCampbell) [inline _data/shared_hosts.yml]: or just a total typo

Comment (philsturgeon) [inline _data/shared_hosts.yml]: Oh yeah 5.5.30 doesn’t exist at all.

This is why we’re working on a automated solution for all this. :) 

-- 
Phil

Comment (GrahamCampbell) [inline _data/shared_hosts.yml]: avs6

**Pull request merged**